servers/republik/modules/crowdfundings/lib/scheduler/owners.js

```diff
@@ -35,7 +35,7 @@ const getMaxEndDate = (now, daysBeforeEndDate) =>
     .endOf('day')
 
 const createBuckets = (now) => [
-  {
+  /* {
     name: 'membership_owner_prolong_notice',
     endDate: {
       min: getMinEndDate(now, 22),
```

Comment:
```suggestion
      min: getMinEndDate(now, DAYS_BEFORE_END_DATE - 1),
```

Reply:
and `min: getMinEndDate(now, 6)` for `T-7`

Reply:
Prepping for reinstating mailings mid December.